update.sh

```diff
@@ -170,7 +170,7 @@ fi
 # restart services
 # If you copy this line make sure to add sudo in front of both commands
 echo "Restarting Mastodon..."
-systemctl reload mastodon-web && systemctl restart mastodon-{sidekiq,streaming}
+systemctl reload mastodon-web && systemctl restart mastodon-{'sidekiq*',streaming}
```

Comment:
Ah, good thing I tested this.
This doesn't catch the default mastodon-sidekiq service.

Reply:
Ah, apparently `status` doesn't list disabled services when enabled ones with the pattern exist.
When the default sidekiq service is enabled, it works.